.github/workflows/retag-release.yaml

```diff
@@ -0,0 +1,53 @@
+# This script adds a "vX.Y.Z" tag to every new release.
+#
+# Our releases are tagged like "1.2.3" but we want people to be able to write
+# GitHub Action workflows to say "uses: readmeio/readme@v1.2.3" because that's
+# the usual GitHub convention.
+
+name: retag-release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  retag-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6.1.0
+        with:
+          script: |
+            const { owner, repo } = context.repo;
```

Comment:
Thoughts on popping this script into a separate JS file? https://github.com/actions/github-script#run-a-separate-file

That way we can lint it, run it locally more easily, etc.

Reply:
Oh cool, didn't notice that. The syntax for doing that is a little janky but I like that idea a lot more than embedding it here in the YAML. New commit coming up shortly…